site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -23,35 +23,39 @@ type: docs
 weight: 600
 ---
 
-The default server configuration is intended for development and testing. When you deploy Polaris in production,
-review and apply the following checklist:
+The default server configuration is intended for development and testing. When you deploy Polaris in production, review
+and apply the following checklist:
+
 - [ ] Configure OAuth2 keys
 - [ ] Enforce realm header validation (`require-header=true`)
 - [ ] Use a durable metastore (JDBC + PostgreSQL)
 - [ ] Bootstrap valid realms in the metastore
 - [ ] Disable local FILE storage
+- [ ] Polaris Server Header
```

Comment:
Why is `Polaris Server Header` a critical point for production configuration?

Reply:
Yes, the `Server` header is off by default due to possible security concerns. If a user wishes to enable it, it is possible. However, it does not look like every user has to make a decision about it.